config/install/field.storage.node.field_weight.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
```

Comment:
Put this file in the `islandora_core_feature` instead of here.  We have a pattern of putting the field storage into the core feature, and putting the field definition for `Repository Item` in `islandora_defaults`. 

...and sorry I didn't catch this earlier when reviewing https://github.com/Islandora-CLAW/islandora/pull/159